package.json

```diff
@@ -55,6 +56,9 @@
     "karma-sourcemap-loader": "^0.3.5",
     "load-grunt-tasks": "^0.4.0",
     "mkdirp": "^0.5.1",
+    "npm-watch": "^0.0.1",
```

Comment:
`npm-watch` is really just a thin wrapper around [nodemon](https://github.com/remy/nodemon) I would rather replace it with the actual nodemon command which would be `nodemon --watch src --exec "npm run build"`

This would mean that the final `serve` command would look like this:

```bash
nodemon --watch src --exec "npm run build" & http-server -p 5678 -c-1 -o
```

and you can get rid of the dependencies on `npm-watch`, `open`, and `parallelshell`